actors/test/commit_post_test.go

```diff
@@ -75,10 +75,12 @@ func TestCommitPoStFlow(t *testing.T) {
 			Method: builtin.MethodsCron.EpochTick,
 			SubInvocations: []vm.ExpectInvocation{
 				{To: builtin.StoragePowerActorAddr, Method: builtin.MethodsPower.OnEpochTickEnd, SubInvocations: []vm.ExpectInvocation{
+					{To: builtin.RewardActorAddr, Method: builtin.MethodsReward.ThisEpochReward},
+					{To: builtin.StoragePowerActorAddr, Method: builtin.MethodsPower.CurrentTotalPower},
 					{To: minerAddrs.IDAddress, Method: builtin.MethodsMiner.OnDeferredCronEvent, SubInvocations: []vm.ExpectInvocation{
+						// XXX-799 this happens twice- see comment in miner_actor.go in processEarlyTerminations function
```

Comment:
You should remove this line but you can reference it in the issue about adding these values as params to OnDeferredCronEvent.

Reply:
done